test/tests/functional/pbs_alps_inventory_check_hook.py

```diff
@@ -51,38 +51,49 @@ class TestAlpsInventoryCheckHook(TestFunctional):
     def setUp(self):
         TestFunctional.setUp(self)
         with open("/etc/xthostname") as xthost_file:
-            self.crayhostname = xthost_file.readline()
-            self.crayhostname = self.crayhostname.rstrip()
+            self.crayhostname = xthost_file.readline().rstrip()
 
         self.server.manager(MGR_CMD_SET, PBS_HOOK,
                             {'enabled': 'true', 'freq': 3},
                             id='PBS_alps_inventory_check')
         self.platform = DshUtils().get_platform()
 
+    def delete_cray_compute_node(self):
+        """
+        Deletes the cray compute node from pbs node list
+        """
+        vnl = self.server.filter(
+            VNODE, {'resources_available.vntype': 'cray_compute'})
+        vlist = vnl["resources_available.vntype=cray_compute"]
+        vnode = vlist[0]
+        rc = self.server.manager(MGR_CMD_DELETE, NODE, id=vnode)
+
     def test_apstat_cmd(self):
         """
         Test the log when apstat is not present in the
-        expected/default location, indicating a Cray system issue.
+        expected/default location, it indicates a Cray system issue.
         """
         now = time.time()
         if self.platform == "craysim":
             os.rename(
                 "/opt/cray/alps/default/bin/apstat",
                 "/opt/cray/alps/default/bin/stat")

```

Comment:
Maybe you should check to see that /opt/cray/alps/default/bin/stat doesn't exist before overwriting it.  What to do if it does exist is a good question.  I guess the answer is either fail the test or skip it.